src/main/webapp/app/exam/manage/exams/exam-checklist-component/exam-checklist.component.ts

```diff
@@ -71,5 +105,65 @@ export class ExamChecklistComponent implements OnChanges, OnInit, OnDestroy {
         this.websocketService.unsubscribe(submittedTopic);
         const startedTopic = this.examChecklistService.getStartedTopic(this.exam);
         this.websocketService.unsubscribe(startedTopic);
+        if (this.longestWorkingTimeSub) {
+            this.longestWorkingTimeSub.unsubscribe();
+        }
+    }
+
+    /**
+     * Evaluates all the quiz exercises that belong to the exam
+     */
+    evaluateQuizExercises() {
+        this.isEvaluatingQuizExercises = true;
+        if (this.exam.course?.id && this.exam.id) {
+            this.examManagementService.evaluateQuizExercises(this.exam.course.id, this.exam.id).subscribe({
+                next: (res) => {
+                    this.alertService.success('artemisApp.studentExams.evaluateQuizExerciseSuccess', { number: res?.body });
+                    this.existsUnassessedQuizzes = false;
+                    this.isEvaluatingQuizExercises = false;
+                },
+                error: (error: HttpErrorResponse) => {
+                    this.dialogErrorSource.next(error.message);
+                    this.alertService.error('artemisApp.studentExams.evaluateQuizExerciseFailure');
+                    this.isEvaluatingQuizExercises = false;
+                },
+            });
+        } else {
+            console.error('Error: Quiz exercises could not be evaluated due to missing course ID or exam ID');
+        }
```

Comment:
Please do not log to the console. Since you added the translations for this I think you wanted to use this.alertService.error?
```suggestion
        }
```

Reply:
I agree, we should not use the console here.